dev/flake.nix

```diff
@@ -15,6 +15,10 @@
         (inputs.git-hooks + /flake-module.nix)
       ];
 
+      flake.om.develop.default.readme = # md
```

Comment:
Why introduce a 2nd omnix configuration? Shouldn't this go in `nix/modules/om.nix`?

Reply:
The `direnv` environment is loaded using `use omnix ./dev`, and the prompt message is taken from the top-level flake output of `dev`.